modules/administration-guide/partials/con_setup-backup-server.adoc

```diff
@@ -0,0 +1,34 @@
+[id="con_setup-backup-server"]
+= Setting up a backup server
+
+The following section describes the supported {prod-short} backup servers and provides information for their setup.
+
+include::partial$snip_internal-backup-server-warning.adoc[]
+
+{prod-short} uses the link:https://restic.readthedocs.io/en/latest/[`restic`] tool to: 
+
+* manage backup snapshots
+
+* push to or to pull backup data from a backup server
++
+[NOTE]
+====
+The `restic` backup tool is licensed under the link:https://opensource.org/licenses/BSD-2-Clause[BSD 2-Clause] license.
```

Comment:
Is this piece of info necessary?

Reply:
Yes, it is... The reason is described [here](https://github.com/eclipse/che/issues/20028).